ENGAGEHF/Dashboard/Survey/FullSurveyView.swift

```diff
@@ -0,0 +1,21 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+struct FullSurveyView: View {
+    var body: some View {
+        Text("Fill in rest of survey here.")
+            .navigationTitle("KCCQ-12 Symptom Survey")
```

Comment:
We should either add the survey here using SpeziQuestionnaire or we should hide this view for now. Might be a separate PR and probably a lower-handing fruit for a small PR that we could do soon.